Removes suggestions

christinach opened the pull request:
closes #121

christinach commented:
@escowles do I want to remove the spellcheck search component as well?

escowles commented:
I think the first thing to do is run the OL test suite against this configuration, and then deploy it to staging for a manual test.

christinach commented:
ok. thanks!

christinach commented:
Locally:
1. I ran the OL test suite and the https://github.com/pulibrary/orangelight/blob/master/spec/features/search_suggestions_spec.rb failed as expected. 
2. I ran it again with the specific spec updated and it passed.

To test it locally on my machine:
1. I changed this line of code: https://github.com/pulibrary/orangelight/blob/bb0fc2ce1f523a9963d32e10a7410804eab4c0b8/lib/tasks/pulsearch.rake#L51
to be: `"https://raw.githubusercontent.com/pulibrary/pul_solr/remove-suggestions/solr_configs/orangelight/#{file}"` and ran the specs for OL successfully.
2. I pointed the OL to the solr core in 8888 that starts from pul_solr and by changing:
https://github.com/pulibrary/orangelight/blob/bb0fc2ce1f523a9963d32e10a7410804eab4c0b8/config/blacklight.yml#L16 to be: `blacklight-core` and the OL tests passed.

I added a branch in OL with the updated spec for when we will merge this PR.

I did not test it in catalog-staging.

@tampakis thank you very much for helping me to upload the documents in solr while I was running the OL against pul_solr and pointing me in the right direction.

tampakis commented on solr_configs/orangelight/conf/solrconfig.xml:
```diff
@@ -434,13 +434,13 @@
         -->
 
-      <str name="spellcheck">true</str>
+      <!-- <str name="spellcheck">true</str>
```
Rather than commenting these out can you remove them?

tampakis commented on solr_configs/orangelight/conf/solrconfig.xml:
```diff
@@ -495,9 +487,9 @@
        </arr>
       -->
-    <arr name="last-components">
+    <!-- <arr name="last-components">
```
One more commented block

christinach replied:
Are you talking only about the 2 lines in 490 or also the rest of the commented blocks from previous commits?

tampakis commented:
Can these lines be removed too? https://github.com/pulibrary/pul_solr/blob/0f21d069fd3625195723df7439b068ed6e09819f/solr_configs/orangelight/conf/schema.xml#L745-L755
https://github.com/pulibrary/pul_solr/blob/0f21d069fd3625195723df7439b068ed6e09819f/solr_configs/orangelight/conf/schema.xml#L861
